src/components/menus/BottomMenu.js

```diff
@@ -10,38 +10,44 @@ export const BottomMenu = ({ menu }) => {
   return (
     <footer className={sty.footer}>
       <Container className={`flex ${sty.containerFooter} `}>
-        <div className={sty.SocialColumn}>
-            <div className={sty.SocialGroup}>
-              <div className={sty.MenuLinkTitle}>Follow Us</div>
-              <ul className={`list-no-style ${sty.SocialLinks}`}>
-                {menu.socials.map((item, index) => (
-                  <li>
-                    <PrismicLink
-                      href={'/'}
-                      key={`socialLink:${index}`}
-                      className={sty.SocialLink}
-                    >
-                      <GatsbyImage
-                        image={item?.social_icon?.gatsbyImageData}
-                        alt={item.social_icon?.alt || ''}
-                      />
-                    </PrismicLink>
-                  </li>
-                ))}
-              </ul>
+        <div className={sty.leftCol}>
```

Comment:
Your classnames here are fine, but just a "in the future" kind of nickpick, personally I prefer (if I can) to name the columns by their content. This is kind of a personal style thing but the reason is that these columns can move around and change, especially going from mobile to desktop so naming them relative to other content (on the left) can be unclear or confusing. But no need to redo them here :)

Reply:
Really good point, didn't think about how names make no sense since it swaps order for mobile, will rename those